README.rst

```diff
@@ -71,10 +71,34 @@ __ `FIWARE Aiakos - Apiary`_
 Top_
 
 
-Running
-=======
+Installation
+============
+
+Using FIWARE package repository (recommended)
+--------------------------------------------
+
+Refer to the documentation of your Linux distribution to set up the URL of the
+repository where FIWARE packages are available (and update cache, if needed).
+Currently, ``http://repositories.testbed.fiware.org/repo/rpm/x86_64``
+
+Then, use the package tool to install ``fiware-aiakos``::
+
+    $ sudo yum install fiware-aiakos
+
+
+Configuration file
+------------------
+
+Although some options can be specified from the command line, as a general rule
+the use of a configuration file is preferable:
+
+- ``/etc/sysconfig/aiakos.yml`` (when running system service)
+- ``{installation_path}/config/aiakos.yml`` (when running manually)
+
+Such configuration file is self-documented, so there you will find a description
+of every configuration option.
 
-After install rpm/deb package, and in order to start the service, run::
+After install and configure the service, and in order to start the service, run::
```

Comment:
After installing and configuring the service, you can execute the service with the following command::